src/notifications/NotificationsManager.ts

```diff
@@ -161,26 +353,210 @@ class NotificationsManager {
    */
   @ready(new notificationsErrors.ErrorNotificationsNotRunning())
   public async sendNotification(
-    nodeId: NodeId,
-    data: NotificationData,
-  ): Promise<void> {
-    const notification: Notification = {
+    {
+      nodeId,
+      data,
+      retries = 64,
+    }: {
+      nodeId: NodeId;
+      data: NotificationData;
+      retries?: number;
+    },
+    tran?: DBTransaction,
+  ): Promise<{
+    notificationId: NotificationId;
+    sendProm: Promise<void>;
+  }> {
```

Comment:
Probably call `sendProm` something like `sendP` or just `p`. We're avoiding using the `prom` name now.

Reply:
this has been done